.github/workflows/autoDIR.yml

```diff
@@ -0,0 +1,29 @@
+name: autoDIR
```

Comment:
# The objective of this GitHub Action is to add a new DIRECTORY.md file to a pull request if needed.
name: directory_writer

Reply:
I'll make this change for the upcoming PR